JCDispatch/build.gradle

```diff
@@ -0,0 +1,76 @@
+plugins {
+    id "application"
+    id "java"
+    id "eclipse"
+    id "idea"
+    id "jacoco"
+}
+
+apply from: "https://raw.githubusercontent.com/kryptnostic/gradles/master/mavenRepos.gradle"
+
+description = "Johnson County Dispatch Data Integrations"
+group = "com.openlattice"
+version = 'v0.0.1'
+
+mainClassName = "com.openlattice.integrations.jcDispatch.Dispatch"
+applicationDefaultJvmArgs = ["-Xmx8G", "-Xms8G", "-server"]
+sourceCompatibility = JavaVersion.VERSION_1_8
+targetCompatibility = JavaVersion.VERSION_1_8
+
+//TODO: Use credentials from environment variables.
+run.args = ["jaildata.csv","username","password"]
+
+configurations {
+    provided
+}
+
+configurations.all {
+    exclude module: 'slf4j-log4j12'
+}
+
+sourceSets {
```

Comment:
The gradle wrapper you are using contains regressions that will prevent me from running it on Windows. Please see my build.gradle in IowaCityDispatchCenter for the missing jar manifest sections.